spec/features/user_can_sign_up_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'web_helpers'
+
+RSpec.feature "Sessions",  type: :feature do
+
+  scenario "signs up the user", :type => :feature do
+    sign_up
+    page.should have_content("Welcome! You have signed up successfully.")
+  end
+
+  scenario "signs in the user", :type => :feature do
+    sign_up
+    visit "/users/sign_out"
```

Comment:
This is not how you should sign out. Click on the devise provided "sign out" link instead. Get requests should not sign you out, it's non restful.

Not sure whether "signs in the user" is the correct name for this feature test?